common/app/common/prices/PriceScope.java

```diff
@@ -0,0 +1,27 @@
+package common.prices;
+
+import com.neovisionaries.i18n.CountryCode;
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.customergroups.CustomerGroup;
+import io.sphere.sdk.models.Reference;
+
+import javax.money.CurrencyUnit;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+public class PriceScope {
```

Comment:
Is it required for the class to have public access? I think it should be in package scope since it is a helper class for PriceFinder.